imports/plugins/included/product-detail-simple/client/containers/productDetailContainer.js

```diff
@@ -112,14 +115,15 @@ class ProductDetailContainer extends Component {
             [oppositeDirection]: "auto",
             [direction]: 0
           }, 600)
-          .delay(4000)
+          .delay(2000)
           .animate({
             [oppositeDirection]: "auto",
             [direction]: -alertWidth
           }, {
             duration: 600,
             complete() {
```

Comment:
Also, could we provide some sort of visual indicator that the "Add to cart" button is disabled? Otherwise I think people might be confused as to why it's not responding